Makefile

```diff
@@ -38,6 +41,7 @@ help:
 	@echo "- dist                    Compile the lib into an ngeo.js standalone build (in dist/)"
 	@echo "- check                   Perform a number of checks on the code"
 	@echo "- lint                    Check the code with the linter"
+	@echo "- less                    Compile less code to css"
```

Comment:
`less code` can mean something different, I think that `less files` is better...